ecl/hqlcpp/hqlsource.cpp

```diff
@@ -7215,17 +7217,22 @@ void FetchBuilder::buildMembers(IHqlExpression * expr)
             translator.buildCsvParameters(instance->nestedctx, tableExpr->queryChild(2), NULL, true);
             unsigned maxColumns = getFieldCount(tableExpr->queryRecord());
             StringBuffer s;
-            s.clear().append("virtual unsigned getMaxColumns() { return ").append(maxColumns).append("; }");
+            s.clear().append("virtual unsigned getMaxColumns() override { return ").append(maxColumns).append("; }");
             instance->classctx.addQuoted(s);
             break;
         }
+        /*
+         * There is no such member - and I'm not sure there needs to be... but this code was written for a reason...
```

Comment:
There is a comment in the thor code:
```
            // NB: the index is based on path iteration matches, so on lookup the elements start at positioned stream
            // i.e. getXmlIteratorPath not used (or supplied) here.
```
So it looks like it may not be needed - although I can imagine some weird xpaths which wouldn't be supported.